variables.ini.dist

```diff
@@ -1,147 +1,142 @@
 [COMMON]
-; Examples of each are listed behind the ;
-SERVER_URL = ; https://www.my-website.com
-LDAP_SERVER = ; ldap://my.ldap.com
-LDAP_SERVER_DN = ; ou=example,dc=org
-TOKEN_EXPIRY_TIME_DAYS = ; 1
-DJANGO_SERVER_URL = ; https://www.my-website.com
-VIRTUALENV_PATH =; /path/virtualenv/atmo
+# Examples of each are listed behind the #
+SERVER_URL = # https://www.my-website.com
+LDAP_SERVER = # ldap://my.ldap.com
+LDAP_SERVER_DN = # ou=example,dc=org
+TOKEN_EXPIRY_TIME_DAYS = 1 # 
```

Comment:
you have put a value for this instead of just replacing the comment marker - is that intentional?